cpp/inc/bond/stream/output_buffer.h

```diff
@@ -298,7 +314,7 @@ class OutputMemoryStream
     {
         if (sizeof(T) * 8 / 7 + _rangeSize + _rangeOffset < _bufferSize)
         {
-            uint8_t* ptr = reinterpret_cast<uint8_t*>(_rangePtr + _rangeSize);
+            char* ptr = _rangePtr + _rangeSize;
```

Comment:
The _pointer_ can be const, but the thing it points to cannot. Not worth the potential confusion. Leaving as-is.